core/model/src/commonMain/kotlin/io/github/droidkaigi/confsched2023/model/DroidKaigi2023Day.kt

```diff
@@ -81,5 +81,17 @@ public enum class DroidKaigi2023Day(
         public fun defaultDyamicThemeDate(): Boolean {
             return Day1.start < Clock.System.now()
         }
+
+        /**
+         * @return appropriate day at this time
+         */
+        fun currentDay(): DroidKaigi2023Day {
+            lateinit var currentDay: DroidKaigi2023Day
+            for (entry in entries.sortedBy { it.day }) {
```

Comment:
Sort once so that it can withstand cases where the definition order is different from the order of event day.